bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/i18n/mercedesme.properties

```diff
@@ -372,10 +372,11 @@ longitudeDescription = Longitude of the location
 
 # thing status types
 
-mercedesme.account.status.authorization-needed = Generate new refresh token
-mercedesme.account.status.refresh-token-missing = Refresh token missing
-mercedesme.account.status.email-missing = EMail missing
-mercedesme.account.status.region-missing = Region missing
-mercedesme.account.status.refresh-invalid = Refresh Interval Invalid
+mercedesme.account.status.config.email-missing = EMail missing
```

Comment:
```suggestion
mercedesme.account.status.config.email-missing = E-mail missing
```

Reply:
I think we had a brief discussion elsewhere, but is should be either "E-mail" or "Email".